.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   build:
     permissions: write-all
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Do people run on ubuntu 20.04? I run 22.04 and - really - people should be up to that by now (24.04 is out, after all).

Reply:
The gh-dxp plugin does not appear to work on teamcity agents out of the box most likely because it is built using new dependencies found on the newer version of linux. We're kind of stuck with 20.04 on the tc agents because that is what jetbrains releases their base image as. 

An alternative would be to build gh-dxp "locally" on the tc agent and use that locally built version in this special case if we want to keep the builder ubuntu-latest for our normal releases.